exercises/practice/largest-series-product/src/test/groovy/LargestSeriesProductSpec.groovy

```diff
@@ -0,0 +1,168 @@
+    }
+
+    @Ignore
+    def "can find the largest product of 3"() {
+        expect:
+        LargestSeriesProduct.largestProduct(digits, span) == expected
+
+        where:
+        digits       | span || expected
+        "1027839564" | 3    || 270
+    }
+
+    @Ignore
+    def "can find the largest product of 5 with numbers in order"() {
+        expect:
+        LargestSeriesProduct.largestProduct(digits, span) == expected
+
+        where:
+        digits       | span || expected
+        "0123456789" | 5    || 15120
+    }
+
+    @Ignore
+    def "can get the largest product of a big number"() {
+        expect:
+        LargestSeriesProduct.largestProduct(digits, span) == expected
+
+        where:
+        digits                                               | span || expected
+        "73167176531330624919225119674426574742355349194934" | 6    || 23520
+    }
+
+    @Ignore
+    def "reports zero if the only digits are zero"() {
+        expect:
+        LargestSeriesProduct.largestProduct(digits, span) == expected
+
+        where:
+        digits | span || expected
+        "0000" | 2    || 0
+    }
+
+    @Ignore
+    def "reports zero if all spans include zero"() {
+        expect:
+        LargestSeriesProduct.largestProduct(digits, span) == expected
+
+        where:
+        digits  | span || expected
+        "99099" | 3    || 0
+    }
+
+    @Ignore
+    def "rejects span longer than string length"() {
+        when:
+        LargestSeriesProduct.largestProduct(digits, span)
+
+        then:
+        IllegalArgumentException exceptionThrown = thrown(IllegalArgumentException)
+        exceptionThrown.message == "span must be smaller than string length"
+
+        where:
+        digits | span
+        "123"  | 4
+    }
+
+    @Ignore
+    def "reports 1 for empty string and empty product (0 span)"() {
+        expect:
+        LargestSeriesProduct.largestProduct(digits, span) == expected
+
+        where:
+        digits | span || expected
+        ""     | 0    || 1
+    }
+
+    @Ignore
+    def "reports 1 for nonempty string and empty product (0 span)"() {
+        expect:
+        LargestSeriesProduct.largestProduct(digits, span) == expected
+
+        where:
+        digits | span || expected
+        "123"  | 0    || 1
+    }
+
+    @Ignore
+    def "rejects empty string and nonzero span"() {
+        when:
+        LargestSeriesProduct.largestProduct(digits, span)
+
+        then:
+        IllegalArgumentException exceptionThrown = thrown(IllegalArgumentException)
+        exceptionThrown.message == "span must be smaller than string length"
+
+        where:
+        digits | span
+        ""     | 1
+    }
+
+    @Ignore
+    def "rejects invalid character in digits"() {
+        when:
+        LargestSeriesProduct.largestProduct(digits, span)
+
+        then:
+        IllegalArgumentException exceptionThrown = thrown(IllegalArgumentException)
+        exceptionThrown.message == "digits input must only contain digits"
```

Comment:
If we're expecting an argument with a particular message, I feel that message should go in the where block as our `expected` value. That keeps consistency with the other test cases which have `digits`, `span`, and `expected`.

Reply:
@BNAndras, thanks for review!

Unified the "where:" blocks in https://github.com/exercism/groovy/pull/437/commits/9c0889c515a813203e27e5f7e4dd0e1f282dcd45